### PR TITLE
Link shared module with iosApp

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: iosApp/Picover.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -227,6 +228,28 @@
...
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/zsh;
+			shellScript = "cd \"$SRCROOT/..\"\n./gradlew :shared:embedAndSignAppleFrameworkForXcode\n";
+			showEnvVarsInLog = 0;
+		};
+/* End PBXShellScriptBuildPhase section */
```
Comment (MarekMacko) [inline iosApp/Picover.xcodeproj/project.pbxproj:251]: Step description [here](https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-integrate-in-existing-app.html#connect-the-framework-to-your-ios-project)

Please note that I've changed shell from default`sh` to `zsh` so please verify if it works for you.

File: iosApp/Picover.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -406,6 +432,11 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
+				OTHER_LDFLAGS = (
+					"$(inherited)",
+					"-framework",
+					shared,
+				);
```
Comment (MarekMacko) [inline iosApp/Picover.xcodeproj/project.pbxproj:439]: That's the result of point 6 from the guide. 
Unfortunately, adding value `$(inherited) -framework shared` via Xcode result in below configuration
```
OTHER_LDFLAGS = "\"$(inherited) -framework shared\"",
```
which doesn't work. 
I've found the correct value by comparing it with the generated project from the template.

File: iosApp/Picover.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -423,6 +454,8 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"picover/Preview Content\"";
 				ENABLE_PREVIEWS = YES;
+				ENABLE_USER_SCRIPT_SANDBOXING = NO;
+				FRAMEWORK_SEARCH_PATHS = "$(SRCROOT)/../shared/build/xcode-frameworks/$(CONFIGURATION)/$(SDK_NAME)";
```
Comment (MarekMacko) [inline iosApp/Picover.xcodeproj/project.pbxproj:458]: `$CONFIGURATION` is Debug or Release

Example:
image

**Comment by gmatyszczak** [comment]
nice work!

**Review by gmatyszczak — APPROVED** [review]

**Pull request merged**